server/imports/startup/index.js

```diff
@@ -15,3 +15,5 @@ import './methods.js'
 
 // Start Cache
 import './cache.js'
+
+async = require('async')
```

Comment:
Put this in the top of the file.  We will also need to refactor existing code to use this global call to avoid needless collisions.